TOC.md

```diff
@@ -11,8 +11,8 @@
   + [基本功能](/basic-features.md)
   + [实验特性](/experimental-features.md)
   + 性能测试报告
-    + [Sysbench 性能对比 - v5.0 对比 v4.0](/benchmark/benchmark-sysbench-v5-vs-v4.md)
-    + [TPC-C 性能对比 - v5.0 对比 v4.0](/benchmark/v5.0-performance-benchmarking-with-tpcc.md)
+    + [Sysbench 性能对比 - v5.1.0 对比 v5.0.2](/benchmark/benchmark-sysbench-v5.1.0-vs-v5.0.2.md)
+    + [TPC-C 性能对比 - v5.1.0 对比 v5.0.2](/benchmark/v5.1-performance-benchmarking-with-tpcc.md)
```

Comment:
@dbsid @qiancai In test reports that compare different TiDB versions, the titles do not indicate the specific patch version, which is advised by @shuke987. 
```suggestion
    + [Sysbench 性能对比 - v5.1 对比 v5.0](/benchmark/benchmark-sysbench-v5.1-vs-v5.0.md)
    + [TPC-C 性能对比 - v5.1 对比 v5.0](/benchmark/v5.1-performance-benchmarking-with-tpcc.md)
```